services/web/server/src/simcore_service_webserver/products/_invitations_handlers.py

```diff
@@ -60,7 +60,7 @@ async def generate_invitation(request: web.Request):
 
     invitation = InvitationGenerated(
         product_name=generated.product,
-        issuer=generated.issuer,  # type: ignore[arg-type]
+        issuer=generated.issuer,
```

Comment:
so that was a bad ignore right?

Reply:
type was overwritten. this is now correct.